src/polymer-cli.ts

```diff
@@ -42,6 +43,21 @@ export class PolymerCli {
   }
 
   run(args) {
+    // If the "--version"/"-v" flag is ever present, just print
+    // the current version. Useful for globally installed CLIs.
+    if (args.indexOf('--version') > -1 || args.indexOf('-v') > -1) {
+      console.log(packageJSON.version);
```

Comment:
I'm a big fan of keeping all of your `require` statements at the top of the file, it's a helpful pattern for when you need to hunt down missing/unused dependencies, circular dependencies in larger applications, etc.